wrappers/gcpckms/gcpckms.go

```diff
@@ -194,9 +193,10 @@ func (s *Wrapper) Type(_ context.Context) (wrapping.WrapperType, error) {
 	return wrapping.WrapperTypeGcpCkms, nil
 }
 
-// KeyId returns the last known key id
+// KeyId returns the last known key id which is determined when the wrappers is
+// configured. See: Wrapper.Config(...)
 func (s *Wrapper) KeyId(_ context.Context) (string, error) {
-	return s.currentKeyId.Load().(string), nil
```

Comment:
Are we okay with changing the contract here? The previous behavior after an Encrypt operation occurred would return the keyId + key version while now we will just be returning the keyId. 

Adding this to the test suite exposes the difference 

```
--- a/wrappers/gcpckms/gcpckms_acc_test.go
+++ b/wrappers/gcpckms/gcpckms_acc_test.go
@@ -65,6 +65,14 @@ func TestGcpCkmsSeal_Lifecycle(t *testing.T) {
                t.Fatalf("err: %s", err.Error())
        }

+       keyId, err := s.KeyId(context.Background())
+       if err != nil {
+               t.Fatalf("err: %s", err.Error())
+       }
+       if swi.KeyInfo.KeyId != keyId {
+               t.Fatalf("expected %s got: %s", keyId, swi.KeyInfo.KeyId)
+       }
+
        pt, err := s.Decrypt(context.Background(), swi)
        if err != nil {
                t.Fatalf("err: %s", err.Error())
```
```
 gcpckms_acc_test.go:73: expected projects/hc-vault-testing/locations/global/keyRings/vault-test-keyring/cryptoKeys/vault-test-key got: projects/hc-vault-testing/locations/global/keyRings/vault-test-keyring/cryptoKeys/vault-test-key/cryptoKeyVersions/1
```

Reply:
Ty for the review and question.  I've pushed a [fixup commit](https://github.com/hashicorp/go-kms-wrapping/pull/211/commits/ea5013c6017c187de2f0370e3a0a17efad491078) that addresses this and adds unit tests to assert what I believe to be the correct behavior.